packages/core/src/routes/session.ts

```diff
@@ -228,6 +230,35 @@ export default function sessionRoutes<T extends AnonymousRouter>(router: T, prov
     }
   );
 
+  router.post(
+    '/session/bind-social',
+    koaGuard({
+      body: object({
+        connectorId: string(),
+      }),
+    }),
+    async (ctx, next) => {
+      const { connectorId } = ctx.guard.body;
+      const { result } = await provider.interactionDetails(ctx.req, ctx.res);
+      assertThat(result, 'session.connector_session_not_found');
+      assertThat(result.login?.account, 'session.unauthorized');
```

Comment:
not sure whether this should be the following code snippet?
```suggestion
      assertThat(result.login?.accountId, 'session.unauthorized');
```

FYI
<img width="617" alt="image" src="https://user-images.githubusercontent.com/15182327/155076642-0aab453f-808a-408d-9365-979d27aa73ec.png">
<img width="655" alt="image" src="https://user-images.githubusercontent.com/15182327/155076770-ada6bf47-2c56-4509-9a2a-cb2cb619324e.png">